src/components/NotFound/NotFound.test.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import { shallow } from 'enzyme';
-import NotFound from './NotFound';
+import NotFound from "./NotFound";
 
-describe('<NotFound/>', () => {
-    it('should has div', () => {
+
+
+describe('<NotFound/> render', () => {
+    it('should render NotFound component', () => {
```

Comment:
there are 2 test more. You should check are there image and text?

Reply:
ok , i`ll do it